yb-voyager/cmd/common.go

```diff
@@ -1227,7 +1226,9 @@ type TargetSizingRecommendations struct {
 	TotalShardedSize   int64
 }
 
-//==========================================//
+var ASSESS_MIGRATION_PAYLOAD_VERSION = "1.0"
```

Comment:
I believe `ASSESS_MIGRATION_PAYLOAD_VERSION` and `AssessMigrationPayload` are YugabyteD control_plane specific so should we move these to `cp` package just to keep things in a place

Reply:
reason i kept it here to keep it together with AssessMigrationPayload.
Once we shift that to the package then we can do this also at same time.